.rubocop.yml

```diff
@@ -26,6 +26,10 @@ Metrics/ClassLength:
 
 Metrics/MethodLength:
   CountAsOne: ['array', 'heredoc', 'hash']
+  Exclude:
+    - app/controllers/api/v1/users_controller.rb
```

Comment:
I wasn't able to find an easy way to minimize the number of lines in the user_params of the User Controller, so I added it to be excluded here. Upon adding it, the methodlength warning went off for several additional controllers, so I added them as well.